NewAndNoteworthy/CHANGELOG-API.md

```diff
@@ -7,6 +7,15 @@ This section describes API removals that occurred in past releases, and upcoming
 # API Changes
 
 Below is the detailed descriptions of API changes and mitigation efforts API consumers need to take.
+## API Changes in CDT 11.4.
```

Comment:
I guess this is a breaking API change, so can be listed here. I recommend it goes in the N&N for 11.4 once I create it in the next few days - like https://github.com/eclipse-cdt/cdt/blob/main/NewAndNoteworthy/CDT-11.3.md#api-changes-current-and-planned

If we are doing something that is a breaking change, can we ditch the reexport too?